public/app/features/alerting/unified/components/rule-editor/labels/LabelsField.tsx

```diff
@@ -248,98 +252,88 @@ export function LabelsWithSuggestions({ dataSourceName }: LabelsWithSuggestionsP
   const { installed: labelsPluginInstalled = false, loading: loadingLabelsPlugin } = usePluginBridge(
     SupportedPlugin.Labels
   );
-  const [selectedKey, setSelectedKey] = useState('');
 
   const { loading, keysFromExistingAlerts, groupedOptions, getValuesForLabel } = useCombinedLabels(
     dataSourceName,
     labelsPluginInstalled,
     loadingLabelsPlugin,
     labelsInSubform,
-    selectedKey
+    ''
```

Comment:
If we are now always passing en empty string, shouldn't we remove this param from the function? I couldn't find any other place where we call this function either.